### PR TITLE
some updates

### PR DESCRIPTION
**Pull request by f4bio**
* versions (converse.js 9.1.0)
* docs
* dockerfile (+multiarch)
* entry points
* nginx config
* gitignore

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
+ARG CONVERSEJS_VERSION=9.1.0
+
+LABEL org.label-schema.schema-version="9.1.0"
+LABEL org.label-schema.name="converse.js/converse.js"
+LABEL org.label-schema.description="messaging freedom"
+LABEL org.label-schema.url="https://conversejs.org/"
+LABEL org.label-schema.application="converse.js"
+LABEL org.label-schema.build-date=$BUILD_DATE
+LABEL org.label-schema.version=$BUILD_VERSION
+LABEL org.opencontainers.image.source="https://github.com/f4bio/converse.js"
```
Comment (jcbrand) [inline Dockerfile]: Could you please change this to https://github.com/conversejs/converse.js-docker ?

According the these [docs](https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys) this is the _URL to get source code for building the image_.

File: Dockerfile [hunk 2]
```diff
@@ -1,47 +1,48 @@
...
+  software-properties-common gnupg gosu curl ca-certificates zip unzip git
+RUN apt-get --yes install python3-dev python3-twisted python3-openssl
+
+RUN curl -sLO \
+  https://github.com/conversejs/converse.js/releases/download/v$CONVERSEJS_VERSION/converse.js-$CONVERSEJS_VERSION.tgz
+RUN tar xzf converse.js-$CONVERSEJS_VERSION.tgz
+RUN mv package/* /usr/share/nginx/html/
+RUN curl -sL \
+  -o /usr/share/nginx/html/index.html \
+  https://raw.githubusercontent.com/conversejs/converse.js/master/fullscreen.html
```
Comment (jcbrand) [inline Dockerfile]: The file `fullscreen.html` being downloaded is hardcoded to use the hosted Punjab instance at https://conversejs.org/http-bind/ (see the `bosh_service_url` setting).

Given that this docker build is including its own Punjab instance, we'll need to use an index.html file that sets `bosh_service_url` to the URL of this build's Punjab instance.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+# Created by https://www.toptal.com/developers/gitignore/api/linux,macos,dotenv,windows,intellij,visualstudiocode
+# Edit at https://www.toptal.com/developers/gitignore?templates=linux,macos,dotenv,windows,intellij,visualstudiocode
+
+### dotenv ###
+.env
+
+### Intellij ###
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
```
Comment (jcbrand) [inline .gitignore]: This `.gitignore` file is huge and includes all kinds of things that one wouldn't expect to be in a checkout of this repo.

IMO developers should have a global `.gitignore_global` that ignores all kinds of things specific to their setup and not put everything inside a project's `.gitignore`.

Could you please only ignore the files that would credibly be inside a checkout of this repo and which we don't want to track?

**Review by jcbrand — CHANGES_REQUESTED** [review]
Hi @f4bio 

Thanks a lot for taking the time to update this repo! :pray: 

Looks good, we can definitely merge this, but I have left a few review comments first.

File: Dockerfile (hunk 2 above)
Comment (f4bio) [reply Dockerfile]: oh. nice! I didn't notice that at all. 
I'm not really a fan of search&replace string in files like this, so I hope this solution is fine by you too?!

**Comment by f4bio** [comment]
> Hi @f4bio
> 
> Thanks a lot for taking the time to update this repo! 🙏
> 
> Looks good, we can definitely merge this, but I have left a few review comments first.

Nah... I have to thank you (& contributors), for providing such an awesome, **open source** lib/tool!

**Pull request merged**